local_ssh.hcl

```diff
@@ -57,7 +57,7 @@ job "ssh-client" {
         port = "http"
 
         check {
-          name = "http-${NOMAD_META_BOX_NAME}-up"
+          name = "http-$${NOMAD_META_BOX_NAME}-up"
```

Comment:
is this supposed to be double $?

Reply:
yes - it's evaluated through terraform, but this is actually nomad templating.  `$${}` is an escape in HCL.